### PR TITLE
Pe 832 support drives folders

### PR DESCRIPTION
**Pull request by agsuy**
Only Docs and one BASH script. Interaction is with BATS docker only.

- Migrates documentation form Docker repository
- Adds new folder to hold BATS scripts

### CODE REVIEW

File: .bats/entities.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd $HOME/ardrive-cli/
+
+cache="$(yarn ardrive list-all-drives -w $WALLET)"
+driveID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .driveId' | cut -d '"' -f 2)"
+folderID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .rootFolderId' | cut -d '"' -f 2)"
```
Comment (matibat) [inline .bats/entities.sh:7]: I tested it in isolation and works well. Although I'd like to get rid of that optional field (i.e. `?`) so we enforce the input to be in good shape
```suggestion
driveID="$(echo $cache | jq --raw-input '[.[] | select(.drivePrivacy == "public")][0] | .driveId')"
folderID="$(echo $cache | jq --raw-input '[.[] | select(.drivePrivacy == "public")][0] | .rootFolderId')"
```
**Explanation**
1. removing the question mark throws an error (cannot index an array with a string value)
2. replacing the recursive operator by explicitly iterating the expected array. `..` -> `.[]` 
3. we get rid of the call to `cut` to remove the quotes by using the `-r --raw-input` flag at `jq`

File: .bats/entities.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+export PUB_DRIVE_ID="$driveID"
+export PUB_FOLD_ID="$folderID"
```
Comment (matibat) [inline .bats/entities.sh:10]: We could replace the above variable declarations for two functions `get_1st_drive_id_from_wallet` and `get_1st_folder_id_from_wallet`. That way we could fancily do the following
```suggestion
export PUB_DRIVE_ID="$(get_1st_drive_id_from_wallet ${WALLET})"
export PUB_FOLD_ID="$(get_1st_folder_id_from_wallet ${WALLET})"
```
But again: it's just to be fancy :nail_care:

File: bats_test/network_tools.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Network tests
+
+Useful to monitor both inbound and outbound traffic to BATS Docker
+
+Disclaimer: *Might now work on MacOS*
```
Comment (matibat) [inline bats_test/network_tools.md:5]: Why so? :/

File: bats_test/readme.md [hunk 4]
```diff
@@ -1,64 +1,33 @@
...
-Some guidelines on writing your own BATS tests
-## Globals
+To run a single file, just use
+``` bats <my-test-file.bats>```
 
-There are several global variables you can use to introspect on Bats tests:
+e.g.
 
-- $BATS_TEST_FILENAME is the fully expanded path to the Bats test file.
+```bats ../test_samples//testing_hooks/hooks_sample.bats```
```
Comment (matibat) [inline bats_test/readme.md:9]: Nit
```suggestion
```bats ../test_samples/testing_hooks/hooks_sample.bats```
```

File: bats_test/wallets.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Interact with a wallet
+
+
+
```
Comment (matibat) [inline bats_test/wallets.md:4]: nit
```suggestion

```

File: bats_test/wallets.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## Wallet Operations
+
+There is a $WALLET variable directly pointing to /home/node/tmp/wallet.json inside the Docker.
+
+In order to run any command that requires a wallet you could just replace its path with $WALLET
+
+e.g. for a private file
+
+``yarn ardrive file-info -f [file-id] -w $WALLET -p [my-unsafe-password]``
```
Comment (matibat) [inline bats_test/wallets.md:39]: The syntaxis of `[...]` in the shell context usually refers to an optional and might be confusing. Consider using an example variable name instead.
```suggestion
``yarn ardrive file-info -f $FILE_ID -w $WALLET -P``
```

Comment (matibat) [reply bats_test/wallets.md:39]: Also, please confirm that the two backticks are what we want. I usually see one or three

File: bats_test/writing_tests.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+[Files](https://github.com/bats-core/bats-file#index-of-all-functions)
+
+[Assertions](https://github.com/bats-core/bats-assert#usage)
+
+
+## Basic Commands
+
+### Log to console
+
+Always  ``` echo 'text' >&3 ```
```
Comment (matibat) [inline bats_test/writing_tests.md:38]: Would you mind specifying which STD stream does `>&3` stand for? Is it the error stream?

File: bats_test/writing_tests.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Only on failures ```echo 'text' ```
```
Comment (matibat) [inline bats_test/writing_tests.md:40]: Something special happens when I log an error like that?

**Review by matibat — DISMISSED** [review]
Please look my comments

File: bats_test/writing_tests.md (hunk 8 above)
Comment (agsuy) [reply bats_test/writing_tests.md:40]: One is only on failures, the other always logs

File: bats_test/writing_tests.md (hunk 7 above)
Comment (agsuy) [reply bats_test/writing_tests.md:38]: Its on BATS documentation. Its a way to log directly https://bats-core.readthedocs.io/en/stable/writing-tests.html#file-descriptor-3-read-this-if-bats-hangs

File: bats_test/network_tools.md (hunk 3 above)
Comment (agsuy) [reply bats_test/network_tools.md:5]: Untested. 
Networking on Mac is a bit limited so I cannot assure everything works there. 
https://docs.docker.com/desktop/mac/networking/

File: .bats/entities.sh (hunk 2 above)
Comment (agsuy) [reply .bats/entities.sh:10]: This is opinionated, gonna keep shorter names for now.

**Review by arielmelendez — APPROVED** [review]

**Comment by agsuy** [comment]
We could iterate on jq command on a follow up but it is not a priority.

**Pull request merged**